applications/ConvectionDiffusionApplication/python_scripts/python_solvers_wrapper_convection_diffusion.py

```diff
@@ -12,31 +12,38 @@ def CreateSolverByParameters(model, solver_settings, parallelism):
         raise Exception("input is expected to be provided as a Kratos Parameters object")
 
     solver_type = solver_settings["solver_type"].GetString()
+    time_integration_method = "implicit / not defined"
+    if solver_settings.Has("time_integration_method"):
+        if solver_settings["time_integration_method"].GetString() == "explicit":
+            time_integration_method = solver_settings["time_integration_method"].GetString()
 
     # Solvers for OpenMP parallelism
     if (parallelism == "OpenMP"):
-        if (solver_type == "transient" or solver_type == "Transient"):
-            solver_module_name = "convection_diffusion_transient_solver"
+        if time_integration_method == "explicit":
+            solver_module_name = "convection_diffusion_explicit_solver"
+        else:
```

Comment:
This must be an `elif` statement. An `else` with an error saying that the provided `time_integration_method` is wrong must be placed below.

Reply:
Ok, even though setting `implicit` by default will never happen. Done.